plugins/modules/jenkins_node.py

```diff
@@ -467,6 +935,41 @@ def main():
             num_executors=dict(type='int'),
             labels=dict(type='list', elements='str'),
             offline_message=dict(type='str'),
+            launch_ssh=dict(
+                type='dict',
+                options=dict(
+                    host=dict(type='str'),
```

Comment:
Shouldn't this one, at least, be required? Or have a default like `localhost`?

Reply:
It's not required to be set in Jenkins, so I thought that it would make sense not to enforce setting it here.

Reply:
ok, fair enough.